src/app/datasets/datasets-filter/datasets-filter.component.spec.ts

```diff
@@ -512,7 +512,7 @@ describe("DatasetsFilterComponent", () => {
           condition: {
             lhs: "",
             rhs: "",
-            relation: "EQUAL_TO_STRING",
+            relation: "CONTAINS_STRING",
```

Comment:
this looks like there is still something missing to _add_ `CONTAINS_STRING` instead of replacing `EQUAL_TO_STRING`